assets/source/setup-guide/app/helpers/effects.js

```diff
@@ -4,6 +4,9 @@
 import { useSelect, useDispatch } from '@wordpress/data';
 import { useEffect, useCallback } from '@wordpress/element';
 
+import { __ } from '@wordpress/i18n';
```

Comment:
Remove empty line in between the imports.

```suggestion
import { useEffect, useCallback } from '@wordpress/element';
import { __ } from '@wordpress/i18n';
```